lib/codegen/EmitConfigParser.cpp

```diff
@@ -280,14 +279,9 @@ static std::pair<llvm::Value *, llvm::BasicBlock *> getEval(KOREDefinition *def,
   case SortCategory::Map:
   case SortCategory::List:
   case SortCategory::Set:
-    if (symbolDecl->getAttributes().count("hook")) {
-      // if this is a hook then we are using the sret abi and the reutrned value
-      // is a pointer. Otherwise we need to store it like Bool/MInt
-      retval = new llvm::BitCastInst(result, llvm::Type::getInt8PtrTy(Ctx), "",
-          creator.getCurrentBlock());
-      break;
-    }
-    // fall through
+    retval = new llvm::BitCastInst(result, llvm::Type::getInt8PtrTy(Ctx), "",
```

Comment:
Is there any difference now between Map/List/Set types and the Int/Float/... types above?